ledger-core/v2/log/logcommon/api_level.go

```diff
@@ -19,8 +19,8 @@ const (
 	InfoLevel
 	WarnLevel
 	ErrorLevel
-	FatalLevel
 	PanicLevel
+	FatalLevel
```

Comment:
It is my understanding that both Panic and Fatal levels are currently undocumented?

Reply:
It may be strange to say, but numbers 0-9 are seem to be undocumented in our project as well ... should we?